apxfr/src/boto

```diff
@@ -0,0 +1,400 @@
+# This file contains credentials and other configuration information needed
+# by the boto library, used by gsutil. You can edit this file (e.g., to add
+# credentials) but be careful not to mis-edit any of the variable names (like
+# "gs_access_key_id") or remove important markers (like the "[Credentials]" and
+# "[Boto]" section delimiters).
+#
+# This file was created by gsutil version 4.50 at 2020-05-15 05:17:17.
+#
```

Comment:
Was this file generated by `gsutil config`? Would that command be more useful than this file?

Reply:
Then I'd have to install gsutil into the container, which I was trying to avoid.